src/CountryPicker.js

```diff
@@ -323,15 +325,18 @@ export default class CountryPicker extends Component {
 
   renderCountryDetail(cca2) {
     const country = countries[cca2]
+    const textStyle = this.disabledCountriesByCode[cca2] ? styles.disabledCountryName : styles.countryName;
```

Comment:
it's better either to create an independent variable, aka `disabledCountry = this.disabledCountriesByCode[cca2]` or make an if else statement in order not to write  this.disabledCountriesByCode[cca2] twice, otherwise the code is not optimal